core/storage/base_model/gae_models.py

```diff
@@ -530,6 +530,27 @@ def get_version(cls, entity_id, version_number):
         # pylint: enable=protected-access
 
     @classmethod
+    def get_multi_versions(cls, entity_id, version_numbers):
+        """Gets model instances for each version specified in version_numbers.
+
+        Args:
+            entity_id: str. ID of the entity.
+            version_numbers: list(int). List of version numbers.
+
+        Returns:
+            list(VersionedModel|None). Model instances representing the given
```

Comment:
In what case will a None value be returned here?

Reply:
@seanlip Nice catch. This never returns None. We check that the exploration has not been deleted before we run this and it returns list(VersionedModel) always. Added a doc string to mention that.

Reply:
Oh wait, it does return None when the model does not exist for a particular version. This ca happen for the exploration stats model. It's better to be kept this way.

Reply:
Per discussion, please investigate and ensure the function is robustly-defined in all cases. Add tests as needed.

Reply:
Done.